llama_stack/models/llama/datatypes.py

```diff
@@ -152,7 +159,120 @@ class ToolParamDefinition(BaseModel):
 
 
 @json_schema_type
-class ToolDefinition(BaseModel):
+class WebSearchTool(BaseModel):
```

Comment:
Can we have a Tool class that mandates the properties? instead of just using BaseModel